### DISCUSSION TITLE
Rephrase "Software Sustainability Actions"

### DISCUSSION THREAD

**Discussion by dschien** [Green-Software-Foundation/sci#179] [SCI feedback]
"All actions that can reduce the carbon emissions of a piece of software fall into one of three categories." This statement needs justification. I don't think is it necessary to make such a strong claim for the purpose of this section.

I suggest softening it to something such as:
"The SCI specification intends to encourages the reduction of carbon emission from software through the actions from these three categories: ... "

**Comment by atg-abhishek** [comment]
@dschien I like this phrasing - once the PRs are merged in during the WG call today. Can you make a new PR for this? Thanks!
